package.json

```diff
@@ -5,13 +5,16 @@
   "private": true,
   "scripts": {
     "lint": "eslint . --cache",
-    "postinstall": "npm run build && npm prune",
+    "postinstall": "npm prune",
     "prestart": "npm run start:mongo",
-    "start": "nodemon --ignore build/ -e js,jsx --exec \"npm run build && nf start\"; npm stop",
+    "start": "scripts/start.sh",
```

Comment:
`"start": "sh scripts/start/sh"` like I was saying in Slack. This is better than having to set modes.